sklearn/model_selection/tests/test_split.py

```diff
@@ -206,24 +207,36 @@ def test_kfold_valueerrors():
 
     # Check that a warning is raised if the least populated class has too few
     # members.
-    y = np.array([3, 3, -1, -1, 2])
+    y1 = np.array([3, 3, -1])
 
-    skf_3 = StratifiedKFold(3)
+    skf_2 = StratifiedKFold(2)
```

Comment:
Maybe keep this as 3, and instead adjust the `y1` accordingly? (or maybe it doesn't matter as long as it raises the correct error message?)